.github/workflows/lint.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Install dependencies
         run: bundle install
       - name: Create mail log file
-        run: mkdir log && touch log/mail.log
+        run: touch log/mail.log
```

Comment:
could we remove this as well from the readme/dockerfile.dev?

Reply:
so I actually just removed the `touch log/mail.log` commands too, they're regular `Logger` objects so will create the file automatically if it's not there (providing the containing directory is present)